### ISSUE TITLE
Disabled Form elements can not be visually distinguished

### ISSUE THREAD

**Issue by MaxTru** [camunda/camunda-modeler#2761]
__Is your feature request related to a problem? Please describe.__

When modeling a Form and disabling a field, it still looks the same way as an enabled field. See screenshot.

![image](https://user-images.githubusercontent.com/42800119/154144322-be3b7219-b8ad-4286-84a9-2f9506721e6f.png)

Personally I feel that there should be a visual differenciation. Compare BPMN, where also the key aspects of what the process does, are indicated visually (e.g., symbol types). Also compare the `required` attribute, which shows an `*` on the form element.

__Describe the solution you'd like__

Have a visual difference of enabled and disabled form elements (e.g., by using a darker grey).

__Describe alternatives you've considered__

n/a

__Additional context__

n/a

**Comment by nikku** [comment]
I tend to agree. This is related to https://github.com/bpmn-io/form-js/issues/215. 

We shall find a way to "preview" the actual form without giving the user the illusion it is actually an editable form. Feedback / input welcome.

**Comment by nikku** [comment]
So you could argue that the current state is a feature. We simply do not show the preview state, but a disabled mock-up of the form to be edited.

**Comment by nikku** [comment]
If you ask me: I'd rather improve the overall form editing experience, i.e. by incorporating [the form playground](https://github.com/bpmn-io/form-js/tree/master/packages/form-js-playground#readme) rather than fine-tuning this.

**Comment by MaxTru** [comment]
I am not entirely sure about the solution, but I agree that certainly we should invest here.

By the way, this issue is the result of serious user testing I did with my partner yesterday night. 
* She wanted to learn what this Camunda is all about. 
* When I opened the form editor to show how we could implement a UserTask, she was impressed (probably the form design was best moment of the demo for her)
* She was confused that the fields are greyed out no matter how we configure them

**Comment by nikku** [comment]
Consider testing with her again using the playground :wink:. The editor alone is simply not enough to convey what the form does / how it shall behave.

**Comment by andreasgeier** [comment]
We are still in an MVP state with Form Editor UX and I agree that at some point we should put that on the agenda. The disabled look and feel of the editing canvas is one of the topics to reconsider and is already listed in the collection of [Form UX improvements](https://github.com/camunda/camunda-modeler-design/issues/17).

At the same time, I agree with @nikku that features like form preview, visibility of input & output data are valuable improvements, too.
